src/services/ServiceMappingService.ts

```diff
@@ -106,31 +112,61 @@ export class ServiceMappingService {
     }
 
     private getQueryParameters(parameters: IOpenAPI3Parameter[] | undefined, models: IModelsContainer): IMethodParameterModel[] {
-        return (
-            parameters?.map((z) => {
-                const parameter = {
-                    name: lowerFirst(z.name),
-                    optional: z.required === undefined ? false : !z.required,
-                    place: ParameterPlace.Query,
-                    isCollection: false,
-                    dtoType: '',
-                    isModel: false
-                };
-
-                if (this.typesGuard.isSimple(z.schema)) {
-                    parameter.dtoType = this.typesService.getSimpleType(z.schema).dtoType;
-                    return parameter;
-                }
-
-                if (this.typesGuard.isReference(z.schema)) {
-                    parameter.dtoType = this.findModel(models, z.schema)?.dtoType ?? '';
-                    parameter.isModel = true;
-                    return parameter;
-                }
+        if (parameters === undefined) {
```

Comment:
maybe more convenient would be !parameters

Reply:
Done